example/src/components/ChatboxWrapper.tsx

```diff
@@ -35,23 +36,14 @@ const ChatboxWrapper: FC<Props> = ({
     (memberships?.size && memberships?.map((m) => m.memberId)?.toArray()) || [];
   const members = hooks.useMembers(memberIds).data;
 
-  const member = new ImmutableMember(currentMember);
+  const member = currentMember;
 
-  const {
-    mutate: sendMessage,
-  }: { mutate: (message: PartialNewChatMessage) => void } = useMutation(
-    MUTATION_KEYS.POST_ITEM_CHAT_MESSAGE,
-  );
-  const {
-    mutate: deleteMessage,
-  }: { mutate: (message: PartialChatMessage) => void } = useMutation(
-    MUTATION_KEYS.DELETE_ITEM_CHAT_MESSAGE,
-  );
-  const {
-    mutate: editMessage,
-  }: { mutate: (message: PartialChatMessage) => void } = useMutation(
-    MUTATION_KEYS.PATCH_ITEM_CHAT_MESSAGE,
-  );
+  const { mutate: sendMessage }: { mutate: SendMessageFunctionType } =
+    useMutation(MUTATION_KEYS.POST_ITEM_CHAT_MESSAGE);
+  const { mutate: deleteMessage }: { mutate: DeleteMessageFunctionType } =
+    useMutation(MUTATION_KEYS.DELETE_ITEM_CHAT_MESSAGE);
+  const { mutate: editMessage }: { mutate: EditMessageFunctionType } =
+    useMutation(MUTATION_KEYS.PATCH_ITEM_CHAT_MESSAGE);
```

Comment:
If you are really motivated, we should import `usePostItemChanMesssage = useMutation(MUTATION_KEYS.POST_ITEM_CHAT_MESSAGE);` directly from query-client so we don't need to type the mutation each time.

Reply:
Yes this would be tackled in https://github.com/graasp/graasp-query-client/issues/239